app/scenes/Talk/index.js

```diff
@@ -149,38 +152,40 @@ class Talk extends Component {
           friction: 7,
           tension: 30,
         }).start(() => {
-          this.setState(
-            Object.assign({}, newState, { incomingTalk: null }),
-            () => {
-              this.state.animValue.setValue(0);
-              this.talkpane.refs.scrollview.setNativeProps({
-                bounces: true,
-                scrollEnabled: true,
-              });
-              this.talkpane.refs.scrollview.scrollTo({ y: 0, animated: false });
-            }
-          );
+          this.setState(Object.assign({}, newState), () => {
+            this.state.animValue.setValue(0);
+            this.talkpane.refs.scrollview.setNativeProps({
+              bounces: true,
+              scrollEnabled: true,
+            });
+            this.talkpane.refs.scrollview.scrollTo({ y: 0, animated: false });
+          });
         });
       }
     );
   };
+
   share = () => {
     const { talk } = this.state;
-    const speakerHandle = talk.speaker.twitter
-      ? '@' + talk.speaker.twitter
-      : talk.speaker.name;
+    const speakers = talk.speakers;
+
+    let speakerHandles = talk.speakers
+      .map(speaker => speaker.twitter || speaker.name)
+      .join(', ');
```

Comment:
One of those cases where always using an `Array<Speaker>` instead of mixing `Speaker | Array<Speaker>` simplifies logic. This is a good rule to follow in general.